mash_scripts/foreman/1.20.0/foreman-plugins-1.20-rhel7-dist.mash

```diff
@@ -7,6 +7,6 @@ multilib = False
 multilib_method = devel
 tag = foreman-plugins-1.20-rhel7-dist
 inherit = True
-strict_keys = False
+strict_keys = True
```

Comment:
This shouldn't be true on plugins?

Reply:
5e743d1e5a0bc68cdded58a76f4b6f2debc7ca37